MAUI/PDF-Viewer/Save-a-Document.md

```diff
@@ -27,4 +27,38 @@ private void SaveDocument()
 {% endhighlight %}
 {% endtabs %}
 
-To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
\ No newline at end of file
+To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
+
+## Flatten annotations and form fields on save
+
+Flattening refers to the process of converting interactive elements, such as annotations and form fields, into a static, non-editable format within a PDF document. The SfPdfViewer allows you to save the annotations and form fields by Flatten using `FlattenOnSave` API.
+
+### Flatten annotation on save
+
+The FlattenOnSave API helps you to flatten the specified annotation in a PDF document. Refer to the following code example:
+
+{% tabs %}
+{% highlight c# %}
+// Obtain the annotation collection using `SfPdfViewer` instance.
+ReadOnlyObservableCollection<Annotation> annotations = PdfViewer.Annotations;
+// Obtain the first annotation in the annotation collection.
+Annotation annotation = annotations[0];
+// set the FlattenOnSave true to flatten the annotation on save
+annotation.FlattenOnSave = true;
+{% endhighlight %}
+{% endtabs %}
+
+### Flatten form field on save
+
+The FlattenOnSave API helps you to flatten the specified formfield in a PDF document. Refer to the following code example:
```

Comment:
<img width="392" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/ba94eb08-c8d7-4cbc-9dbd-6055a040b17c">

Please apply feedback in other places also.